pkg/util/env.go

```diff
@@ -24,22 +24,31 @@ const fluentdImage = "FLUENTD_IMAGE"
 const esMasterNodeRequestMemory = "ES_MASTER_NODE_REQUEST_MEMORY"
 const esIngestNodeRequestMemory = "ES_INGEST_NODE_REQUEST_MEMORY"
 const esDataNodeRequestMemory = "ES_DATA_NODE_REQUEST_MEMORY"
+const grafanaEnabled = "GRAFANA_ENABLED"
 const grafanaRequestMemory = "GRAFANA_REQUEST_MEMORY"
 const grafanaDataStorageSize = "GRAFANA_DATA_STORAGE"
+const promEnabled = "PROMETHEUS_ENABLED"
 const prometheusRequestMemory = "PROMETHEUS_REQUEST_MEMORY"
 const prometheusDataStorageSize = "PROMETHEUS_DATA_STORAGE"
+const kibanaEnabled = "KIBANA_ENABLED"
 const kibanaRequestMemory = "KIBANA_REQUEST_MEMORY"
 const esDataNodeReplicas = "ES_DATA_NODE_REPLICAS"
 const esIngestNodeReplicas = "ES_INGEST_NODE_REPLICAS"
 const esIngestNodeReplicasDefault = 1
 const esMasterNodeReplicas = "ES_MASTER_NODE_REPLICAS"
+const esEnabled = "ES_ENABLED"
 const esMasterNodeReplicasDefault = 3
 const esDataStorageSize = "ES_DATA_STORAGE"
 const esDataNodeReplicasDefault = 2
 
 const accessControlAllowOrigin = "ACCESS_CONTROL_ALLOW_ORIGIN"
 const sharedVMIDefault = "USE_SYSTEM_VMI"
 
+// GetPrometheusEnabled returns true if Prometheus is enabled for the install
+func GetPrometheusEnabled() bool {
```

Comment:
should the default be true for these new VMI bools?  If not then the install has to explicitly set each one to true, except in the managed install case where you are disabling them.  Not sure it makes much of a difference.

Reply:
That's handled in the charts.  The default values in the charts are true, to be overridden by the CR and/or profile values.